tests/functional/ui/test_cephfs_subvolume_metrics.py

```
@@ -293,3 +296,144 @@ def test_cephfs_subvolume_metric_units_and_labels(self, setup_ui_class):
+            constants.CEPHFS_SUBVOLUME_MAX_TOP_10_ROWS,
+        )
+        subvolume_metrics_card.wait_for_row_count(
+            expected_count=constants.CEPHFS_SUBVOLUME_MAX_TOP_10_ROWS,
+            timeout=360,
+            sleep=20,
+        )
+
+    @tier2
+    @ui
+    @pytest.mark.parametrize(
+        argnames=["metric"],
```

Ruff (0.15.20)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `str`

(PT006)